srcpkgs/mesa-asahi/template

```diff
@@ -0,0 +1,181 @@
+# Template file for 'mesa-asahi'
+pkgname=mesa-asahi
+version=24.2.0
+revision=1
+_asahiver=20240727
+build_style=meson
+build_helper=qemu
+_llvmver=18
+#Disable LTO flag should be present, see https://gitlab.freedesktop.org/mesa/mesa/-/issues/6911
+configure_args="-Dglvnd=enabled -Dshared-glapi=enabled -Dgbm=enabled -Degl=enabled
+ -Dosmesa=true -Dgles1=enabled -Dgles2=enabled -Dglx=dri -Ddri3=enabled
+ -Dlmsensors=enabled -Dplatforms=x11$(vopt_if wayland ,wayland)
+ -Dllvm=enabled -Db_lto=false -Dcpp_std=gnu++17"
+hostmakedepends="gettext flex pkg-config python3-Mako glslang llvm${_llvmver}
+ $(vopt_if wayland 'wayland-protocols wayland-devel')"
+makedepends="elfutils-devel expat-devel libXdamage-devel
+ libXxf86vm-devel libdrm-devel libffi-devel libva-devel
+ libvdpau-devel libxshmfence-devel ncurses-devel zlib-devel
+ $(vopt_if wayland 'wayland-devel wayland-protocols') llvm${_llvmver}-devel libsensors-devel
+ libXrandr-devel libglvnd-devel libzstd-devel libxml2-devel lua53-devel
+ libarchive-devel python3-pycparser"
+depends="libglvnd"
+short_desc="Open source implementation of OpenGL and Vulkan - Asahi"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="MIT, LGPL-2.1-or-later"
+homepage="https://www.mesa3d.org/"
+changelog="https://docs.mesa3d.org/relnotes.html"
+distfiles="https://gitlab.freedesktop.org/asahi/mesa/-/archive/asahi-${_asahiver}/mesa-asahi-${_asahiver}.tar.gz"
+checksum=af3370162a312c7b8cc5567e5cfa6cba1b59f432f16b3f89770cf6f375c0cfcf
+conflicts="mesa"
```

Comment:
all of the `conflicts` should probably be `replaces`

Reply:
Done.